pcdsdevices/epics_motor.py

```diff
@@ -177,17 +177,27 @@ def format_status_info(self, status_info):
 """
 
     @property
-    def limits(self):
+    def limits(self) -> tuple[float, float]:
         """Override the limits attribute"""
         return self._get_epics_limits()
 
-    def _get_epics_limits(self):
+    def _get_epics_limits(self) -> tuple[float, float]:
         limits = self.user_setpoint.limits
         if limits is None or limits == (None, None):
             # Not initialized
             return (0, 0)
         return limits
 
+    @limits.setter
+    def limits(self, lims: tuple[float, float]):
+        """
+        Write to the epics limits on setattr
+
+        Expects a tuple of two elements, the low and high limits to set.
+        """
+        self.low_limit_travel.put(min(lims))
+        self.high_limit_travel.put(max(lims))
```

Comment:
I know this is an entirely logical assumption to make, but are we sure we can be certain of this?  We don't have any whacky motors that have the limits switched and would require the order to matter?

(a genuine question, I could see some zany IOC that swaps things around for some unreasonable reason)

Maybe my imagination is getting the better of me

Reply:
That line of questioning reminds me - it won't be a problem for `high_limit_travel < low_limit_travel` on the IOC side as you've implied but there _is_ a potential issue there:
You can't set a high-limit below the low limit, and you can't set the low limit above the current high limit. That needs some special handling to my recollection.

Reply:
I think we're guaranteed this by the way the IOC works, but it's a good question

Reply:
oh no you're so right Ken, there's some awful stateful behavior here depending on the original limit settings

Reply:
Glad I commented then, despite the haziness... Fix looks good, I think